packages/extension-chakra-storefront/src/components/order-summary/index.jsx

```diff
@@ -316,7 +303,7 @@ const OrderSummary = ({
                                         </Text>
                                         {!basket.orderNo && (
                                             <Button
-                                                variant="link"
+                                                variant="ghost"
```

Comment:
To match the original, let's have this as a red link. Otherwise, currently Remove appears like a regular text.

```suggestion
                                                variant="link-red"
```

Reply:
oh good catch, it is fixed now